src/components/VrxMenu/VrxMenu.mdx

```diff
@@ -0,0 +1,33 @@
+import {Meta} from "@storybook/blocks";
+
+<Meta title="VrxMenu/About" />
+
+# VireoXmenu
+
+VireoXmenu is a series of components that can be used to create a menu system for your web application.
+It is designed to be used with the component from VireoX library, but can be used independently.
```

Comment:
It is designed to be used with the **components** from....., but can **also** be used independently.